cfg/admin.txt

```diff
@@ -13,3 +13,4 @@ giacomand
 rockdtben
 MrStonedOne
 Jordie0608
+Swankcookie
```

Comment:
yeah, I know, i accidentally created this branch from the first branch i made, when i was trying to figure out how to selfmin. I'm already in the other one.  I can remove myself from this one if you want, but i was told it didn't matter, anyway.